test/e2e/extensions.test.ts

```diff
@@ -1,12 +1,21 @@
 import { describe, test } from "./baseFixture"
 
-describe("Extensions", true, () => {
+function runTestExtensionTests() {
```

Comment:
👍 This allows us to easily run the extension test with various code-server environments (i.e. different CLI flags)